### ISSUE TITLE
Group registration: identifcation of registered endpoints

### ISSUE THREAD

**Issue by chrysn** [core-wg/resource-directory#48]
With the new EP and GP lookup link-format bodies (solving #21), the payload of a group creation has become odd: They are now the only place where a `<>;ep=...` can occur.

From how we express the model in link-format in other places, IMO the cleanest way of creating a group with members would be to address the members by their respective registration resources -- ie. if

~~~
POST /rd?ep=node1 → Created /reg/11
POST /rd?ep=node2 → Created /reg/12
~~~

then the group that contains both would be constructed with

~~~
POST /rd-group?gp=grp-of-some
</reg/11>,</reg/12>
~~~

which is consistent with group lookups, but requires a group manager that only knows the group members by their endpoint names to look them up first through the EP lookup interface.

The alternatives are to keep things as they are (and spell out that the hrefs are ignored and the ep= attributes are used like EP query attributes), or to come up with something that is not direct link-format at all (possibly `</rd-lookup?ep=node1>,</rd-lookup?et=some-type>` to create a group with all nodes of some-type and the node named node1).

My personal preferences are (in decreasing order) `</reg/11>,</reg/12>`, something-different and keep-as-is, but it's not a very strong preference.

**Comment by chrysn** [comment]
(Meeting context: I had planned to mention this when talking about how we could refer to registered endpoints and groups, but there was a common understanding so quickly that I forgot about it.)

**Comment by chrysn** [comment]
Closed in 2155506ef and 8f7ab88f
